src/components/editor/ResourceURIMessage.jsx

```diff
@@ -1,24 +1,20 @@
-// Copyright 2018 Stanford University see LICENSE for license
+// Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 
 // Renders the resource URI message after Save & Publish
 const ResourceURIMessage = (props) => {
-  if (!props.show) {
+  if (!props.show || !props.uri) {
```

Comment:
~~stockholm~~ rubocop syndrome has me looking for places to use guard clauses

Reply:
I like this change.  And the humor.

Reply:
I like this change. I started here, as my mind is likewise in ruby guard clauses, but didn't see it often in examples...